backend/src/main/java/com/example/cms/ticket/TicketController.java

```diff
@@ -0,0 +1,46 @@
+package com.example.cms.ticket;
```

Comment:
I think we will also need `getTicket` method for the "chat" view, but it's OK if it''s marked as separate task

Reply:
You can always use filters to get single ticket